ansible/monitoring.yml

```diff
@@ -39,7 +39,7 @@
         init_config:
         instances:
           - name: API
-            url: "http://localhost:3013/v1/top"
+            url: "http://localhost:3013/v2/top"
```

Comment:
should we keep the `vN` in some central place?

Reply:
Good point, After we merge #17 I'll rebase and use the variable.

Reply:
see dd59135